src/main/scala/mesosphere/marathon/storage/repository/DeploymentRepositoryImpl.scala

```diff
@@ -136,7 +136,7 @@ class DeploymentRepositoryImpl[K, C, S](
   override def ids(): Source[String, NotUsed] = repo.ids()
 
   override def all(): Source[DeploymentPlan, NotUsed] =
-    repo.ids().mapAsync(Int.MaxValue)(get).collect { case Some(g) => g }
+    repo.ids().mapAsync(8)(get).collect { case Some(g) => g }
```

Comment:
Why is it a literal here but not in #5676?

Reply:
👍  Could you please introduce a const instead and use it here?

Reply:
introduced `RepositoryConstants.maxConcurrency`